bazel/browsers/update-script/browser-artifact.ts

```diff
@@ -37,14 +36,20 @@ export class BrowserArtifact {
  * example with `.tar.gz`, we will need to keep track of known extensions
  * and start looking with the first dot/period we discover.
  */
-function getArtifactExtension(filePath: string) {
+export function detectArtifactExtension(filePath: string) {
   let tmpPath: string = filePath;
   let extension: string = '';
   let currentPart: string = '';
 
   // Iterate from the end of the path, finding the largest possible
   // extension substring, accounting for cases like `a/b.tmp/file.tar.gz`.
   while ((currentPart = path.extname(tmpPath)) !== '') {
+    // An extension needs to be a continuous set of alphanumeric characters. This is a strict
+    // requirement, but sufficient for our needs and helps making the determinationmore proof.
```

Comment:
I am not sure what you mean at the end of this sentence

Reply:
updated the comment and added an example